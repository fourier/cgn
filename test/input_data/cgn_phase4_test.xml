<?xml version="1.0" encoding="UTF-8"?>
<!-- default namespace for elements: cgn -->
<root xmlns="https://github.com/fourier/cgn" 
      xmlns:cgn="https://github.com/fourier/cgn"
      xmlns:jcgn="https://github.com/fourier/cgn/java">

  <!-- this test tests the 4th transformation phase -->

  <cgn:copyright>
    Copyright (c) 2014 by My Company
    This file and its contents are Confidential.
  </cgn:copyright>

  <cgn:author>John Doe</cgn:author>

  <!-- test if the field cgn:type is not set, use the default (string) -->
  <objects cgn:package="com.test.phase4.field.type.not.set"
           cgn:read-only="false"
           cgn:json="true"
           cgn:is-set="true">
    <object cgn:name="Test1">
      <field cgn:name="some-field"/>
      <field cgn:name="some-other-field" cgn:type="int"/>
      <field cgn:name="yet-another-field" cgn:type="double"/>
    </object>
  </objects>
  
</root>

