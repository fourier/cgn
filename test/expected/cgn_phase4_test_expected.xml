<?xml version="1.0" encoding="UTF-8"?>
<!-- default namespace for elements: cgn -->
<root xmlns="https://github.com/fourier/cgn"
      xmlns:cgn="https://github.com/fourier/cgn"
      xmlns:jcgn="https://github.com/fourier/cgn/java"><!-- this test tests the 4th transformation phase -->
   <cgn:copyright>
    Copyright (c) 2014 by My Company
    This file and its contents are Confidential.
  </cgn:copyright>
   <cgn:author>John Doe</cgn:author>
   <!-- test if the field cgn:type is not set, use the default (string) -->
   <objects cgn:package="com.test.phase4.field.type.not.set"
            cgn:read-only="false"
            cgn:json="true"
            cgn:is-set="true"
            cgn:copyright="&#xA;    Copyright (c) 2014 by My Company&#xA;    This file and its contents are Confidential.&#xA;  "
            cgn:author="John Doe">
      <object cgn:name="Test1"
              cgn:copyright="&#xA;    Copyright (c) 2014 by My Company&#xA;    This file and its contents are Confidential.&#xA;  "
              cgn:author="John Doe"
              cgn:package="com.test.phase4.field.type.not.set"
              cgn:read-only="false"
              cgn:json="true"
              cgn:is-set="true">
         <field cgn:name="some-field" cgn:type="string"/>
         <field cgn:name="some-other-field" cgn:type="int"/>
         <field cgn:name="yet-another-field" cgn:type="double"/>
      </object>
   </objects>
</root>
